### PR TITLE
Factorized embedding

### PR DESCRIPTION
**Pull request by adamgayoso**
This allows to plug and play with factorizing the donor embeddings

### CODE REVIEW

File: src/scvi_v2/_module.py [hunk 1]
```diff
@@ -74,9 +80,18 @@ def __call__(self, u: NdArray, sample_covariate: NdArray, training: Optional[boo
...
+        if self.n_factorized_embed_dims is None:
+            A_s = nn.Embed(self.n_sample, self.n_latent * self.n_latent, embedding_init=_normal_initializer)(
+                sample_covariate
+            )
+        else:
+            A_s = FactorizedEmbedding(
+                self.n_sample,
+                self.n_latent * self.n_latent,
+                self.n_factorized_embed_dims,
+                embedding_init=_normal_initializer,
```
Comment (justjhong) [inline src/scvi_v2/_module.py:92]: nit give this a name="A_s" so we can access the factors

Comment (justjhong) [reply src/scvi_v2/_module.py:92]: I did this in my xarray pr for the normal embedding

Comment (adamgayoso) [reply src/scvi_v2/_module.py:92]: Wait I'm not following, can you suggest a change?

Comment (adamgayoso) [reply src/scvi_v2/_module.py:92]: got it

**Review by justjhong — APPROVED** [review]

**Pull request merged**